nansat/mappers/mapper_sentinel1_l1.py

```diff
@@ -463,7 +463,7 @@ def read_annotation(self, annotation_files):
 
         return data
 
-    def _init_empty(self, manifest_data, annotation_data):
+    def _init_empty(self, manifest_data, annotation_data,filename):
```

Comment:
Can you add a space before `filename`?

Reply:
done

Reply:
Also update the docstring
```suggestion
    def _init_empty(self, manifest_data, annotation_data, filename):
```